### DISCUSSION TITLE
Prometheus monitoring

### DISCUSSION THREAD

**Discussion by benbgg** [akvorado/akvorado#1329] [Q&A]
I have successfully installed and configured Akvorado using docker compose (it's great, thanks!). 
I would like to scrape metrics using an existing prometheus setup - as far as I can see, it needs to scrape the target ['ip of orchestrator:8081'], path: /api/v0/metrics. 
Does anyone have an example config they might be willing to share?
Thanks in advance.

**Comment by miqueet** [comment]
curious if you ever found an answer to this @benbgg

**Reply by benbgg** [reply]
Afraid not, to be honest I disappeared down a bunch of other rabbit holes; although it looks as though that link still works and still provides some metrics.

It also appears that you also have Alloy in play and I think where I got stuck the last time was trying to only start alloy and have that write to my central prometheus server. When I get a chance I'll take a look - maybe you could just do a custom docker-compose-custom and include that to just start the alloy container.

**Comment by bogi788** [comment]
Assuming you already have the akvorado's API port exposed the prometheus scrape config should be something like this (if scraping from localhost):

```yaml
- job_name: akvorado
  metrics_path: /api/v0/metrics
  static_configs:
    - targets: ["127.0.0.1:8080"]
```

(You'll have to expose and scrape different ports if you want to monitor all akvorado components.)

**Reply by vincentbernat** [reply]
You can use `/api/v0/inlet/metrics`, `/api/v0/outlet/metrics`, `/api/v0/console/metrics`, `/api/v0/orchestrator/metrics`.
